.github/workflows/ci.yaml

```diff
@@ -199,13 +202,6 @@ jobs:
         Add-Content ${{ github.workspace }}/triplets_overlay/x64-osx-release.cmake `
           -Value "set(VCPKG_BUILD_TYPE release)","set(VCPKG_OSX_DEPLOYMENT_TARGET 10.15)"
 
-    # NOTE: Avoids a libtorrent ABI issue. See https://github.com/arvidn/libtorrent/issues/4965
```

Comment:
@glassez You know that I'm back now, but you couldn't at least ping me before merging this, or at least included a link to this in here: https://github.com/orgs/qbittorrent/teams/bug-handlers/discussions/13/comments/4? I would have liked to review this...

Case in point, I don't think this change is good. The workaround you found was to always force passing `-DCMAKE_CXX_STANDARD=17` to everything, but this is not a good practice in CMake. It should only be done if needed, and thus should remain exclusive to the macOS build.

Another disadvantage is that it makes our portfile deviate more from upstream than necessary. We don't want to deviate from upstream too much. In fact, it is in our best interest to contribute back, so that we may benefit from the efforts of others and integrate upstream fixes more easily. It's not like we are hyper-specialized users of libtorrent.

Feel free to submit a hotfix. if you don't have the time for that, I'll get round to it once I've finished catching up with current happenings, or maybe someone else can do it in the meantime.

Reply:
Sorry, it should have been merged a long time ago. I just forgot about it.
>Case in point, I don't think this change is good. The workaround you found was to always force passing -DCMAKE_CXX_STANDARD=17 to everything, but this is not a good practice in CMake. It should only be done if needed, and thus should remain exclusive to the macOS build.

I agree. But I didn't want to be perfect at it. I just wanted to avoid an obstacle in the form of vcpkg, so that CI would continue to work.
> if you don't have the time for that, I'll get round to it once I've finished catching up with current happenings

I'll leave it to you.